gtsam/geometry/Similarity2.cpp

```diff
@@ -15,21 +15,21 @@
  * @author John Lambert
  */
 
-#include <gtsam/geometry/Similarity2.h>
-
-#include <gtsam/geometry/Rot3.h>
-#include <gtsam/geometry/Pose2.h>
 #include <gtsam/base/Manifold.h>
+#include <gtsam/geometry/Pose2.h>
+#include <gtsam/geometry/Rot3.h>
+#include <gtsam/geometry/Similarity2.h>
 #include <gtsam/slam/KarcherMeanFactor-inl.h>
 
 namespace gtsam {
 
 using std::vector;
 
-namespace {
+namespace internal {
```

Comment:
thanks for the PR @varunagrawal. Appreciate you taking the time to help out here.

Was curious the rationale for the `internal` namespace (why its needed and the naming)

Reply:
I followed Frank's suggestion from your PR.

Reply:
I am assuming @varunagrawal this is because you need Sim3 for your A-> B :-)

Reply:
Yup exactly :)